hack/generate-bindata.sh

```diff
@@ -64,8 +66,10 @@ rm -f "${BINDATA_OUTPUT}.tmp"
 
 # These are files for runtime code
 BINDATA_OUTPUT="pkg/generated/bindata.go"
+# IMPORTANT: if you make any changes to these arguments, you must also update
+# pkg/generated/BUILD.
```

Comment:
I looked at that file and I don't see the link?

Reply:
good point, updated this comment to point to `build/bindata.bzl` as well.